.github/workflows/diff-change-to-dist.yaml

```
@@ -42,6 +42,7 @@ jobs:
           github-token: ${{secrets.GITHUB_TOKEN}}
           script: |
             const fs = require('fs').promises
```

Not a big one but Node.js 14 added `fs/promises` directly

```suggestion
            const { readFile } = require('fs/promises')
```

Although we don't do it (optional) it's good to prefix core packages with `node:` in Node.js 16+ too

```suggestion
            const { readFile } = require('node:fs/promises')
```